components/modules/Kconfig

```diff
@@ -14,8 +14,16 @@ menu "NodeMCU modules"
             Includes the bit module. This module provide bit manipulation
             functions on Lua numbers.
 
+    config NODEMCU_CMODULE_BLE
+        bool "Bluetooth GAP/GATT interface module"
+        default "n"
+	select NODEMCU_CMODULE_STRUCT
+        select BT_ENABLED
+        help
+            Includes the simple Bluetooth GAP/GATT module. You must enable the Nimble BLE module as well.
```

Comment:
Can we not `select WHATEVER_NIMBLE_SYMBOL` here, rather than mention it in the help text?

Reply:
Unfortunately not (at least with the version of Kconfig that Espressif uses). The BT_NIMBLE_ENABLED is a *choice* of BT_HOST -- and you can't select a choice. This is a typical example of somebody complaining about it: https://github.com/zephyrproject-rtos/zephyr/issues/6948

Since we can't modify the Espressif supplied IDF, I think we have to select it by hand. It does appear that some versions of Kconfig use the last default setting for a choice (rather than the first). Maybe I can get espressif to adopt that patch.